yt_dlp/extractor/googledrive.py

```diff
@@ -289,55 +295,93 @@ def add_source_format(urlh):
 
 class GoogleDriveFolderIE(InfoExtractor):
     IE_NAME = 'GoogleDrive:Folder'
-    _VALID_URL = r'https?://(?:docs|drive)\.google\.com/drive/folders/(?P<id>[\w-]{28,})'
+    _VALID_URL = r'https?://(?:docs|drive)\.google\.com/drive/(?:folders/(?P<id>[\w-]{28,})|my-drive)'
```

Comment:
```suggestion
    _VALID_URL = r'https?://(?:docs|drive)\.google\.com/drive/(?:folders/(?P<id>[\w-]{28,})|(?P<id>my-drive))'
```
maybe also capture `my-drive` as id?

Reply:
`A subpattern name must be unique`